core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -486,6 +486,17 @@ default void renameColumn(ConnectorSession session, ConnectorTableHandle tableHa
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming columns");
     }
 
+    /**
+     * Add the specified field, potentially nested, to a row.
+     *
+     * @param fieldPath path to a field within the column, including leaf field name.
```

Comment:
> within the column

we don't provide column name as a separate parameter, so it's not "within column"
it's rather "starting with column name"